networkx/algorithms/time_dependent.py

```diff
@@ -0,0 +1,118 @@
+"""Time dependent algorithms."""
+
+from datetime import datetime
+
+import networkx as nx
+from networkx.utils import not_implemented_for
+
+__all__ = ["cd_index"]
+
+
+@not_implemented_for("undirected")
+@not_implemented_for("multigraph")
+def cd_index(G, node, time_delta=5, weight=None):
+    r"""Compute the CD index.
```

Comment:
> I might have the terminology mixed up for this, but is the CD index for the graph? or for the node of the graph? It seems like the index would be different for each choice of node, so it could be described as

You are right! The CD index is calculated for the node of the graph. We have updated the doc_strings so as to be better explained.